src/fluree/db/memorydb.cljc

```diff
@@ -0,0 +1,658 @@
+(ns fluree.db.memorydb
+  (:require [fluree.db.dbproto :as dbproto]
+            [fluree.db.query.schema :as schema]
+            [fluree.db.constants :as const]
+            [fluree.db.flake :as flake #?@(:cljs [:refer [Flake]])]
+            [fluree.db.util.async :refer [<? go-try merge-into?]]
+            #?(:clj  [clojure.core.async :refer [go <!] :as async]
+               :cljs [cljs.core.async :refer [go <!] :as async])
+            [fluree.db.session :as session]
+            [fluree.db.util.async :as async-util]))
+
+(declare bootstrap-flakes genesis-ecount)
+
+(defn fake-conn []
+  "Returns a fake connection object that is suitable for use with the memorydb if
+  no other conn is available."
+  {:transactor? false})
```

Comment:
Can this just be a `def`?

Reply:
@cap10morgan It could have been. It will be expanded to allow options passed in - for now I did a very basic one but did the fn in the mindset of that expansion.